engine/gameobject/src/dmsdk/gameobject/gameobject.h

```diff
@@ -528,6 +528,16 @@ namespace dmGameObject
      */
     HCollection GetCollection(HInstance instance);
 
+    /*#
+     * Retrieve a collection by socket name hash
+     * Note: in native extensions, the register can be retrieved during init using dmEngine::GetGameObjectRegister(dmExtension::AppParams *params)
+     * @name GetCollectionByHash
+     * @param regist [type: dmGameObject::HRegister] Register
+     * @param socket_name [type: dmhash_t] Socket name
+     * @return collection [type: dmGameObject::HCollection] The collection the specified instance belongs to
+     */
+    HCollection GetCollectionByHash(HRegister register, dmhash_t socket_name);
```

Comment:
Two issues:
* `register` is a keyword. We use `regist` as a workaround. See prior code.
* For a function that is "moved",  I'd expect a removal somewhere?